### PR TITLE
No commercial work to be completed by students at our workshops

### PR DESCRIPTION
**Pull request by KimberleyCook**
I noticed this wasn't in our student guide when it should be so have added `Do not bring along and ask for help with any commercial work. This involves work you're being paid to do.` 

Not 100% sure on the wording of this though, @matyikriszta what do you think?

### CODE REVIEW

File: app/views/dashboard/participant_guide.html.haml [hunk 1]
```diff
@@ -28,7 +28,9 @@
           %li
             = raw t('participant_student_guide.attendance_policy.rule5')
           %li
-            = t('participant_student_guide.attendance_policy.rule6')
+            = raw t('participant_student_guide.attendance_policy.rule6')
```
Comment (despo) [inline app/views/dashboard/participant_guide.html.haml:31]: As rule6 doesnt contain ant HTML `raw` is not needed here

**Review by despo — CHANGES_REQUESTED** [review]
👍

File: app/views/dashboard/participant_guide.html.haml (hunk 1 above)
Comment (KimberleyCook) [reply app/views/dashboard/participant_guide.html.haml:31]: oh that's what it means, cheers. Will update :)

Comment (despo) [reply app/views/dashboard/participant_guide.html.haml:31]: Thanks @KimberleyCook !

Comment (KimberleyCook) [reply app/views/dashboard/participant_guide.html.haml]: @despo updated.

Comment (despo) [reply app/views/dashboard/participant_guide.html.haml]: @KimberleyCook looks good. Can you rebase please!

**Review by despo — APPROVED** [review]
LGTM 💯

**Pull request merged**